test/nginx_system_test.sh

```diff
@@ -1101,6 +1103,47 @@ start_test Base config has purging disabled.  Check error message syntax.
 OUT=$($WGET_DUMP "$HOSTNAME/pagespeed_admin/cache?purge=*")
 check_from "$OUT" fgrep -q "pagespeed EnableCachePurge on;"
 
+start_test Default server header in html flow.
+URL="http://$SECONDARY_HOSTNAME/mod_pagespeed_example/"
+HEADERS="--header=Host:headers.example.com"
+OUT=$($WGET_DUMP -O /dev/null -S $HEADERS $URL 2>&1)
```

Comment:
Could you write this test with `http_proxy=` instead?  I know we have some of both and they're technically equivalent, but we've been trying to standardize some.

So:

```
URL=http://headers.example.com/mod_pagespeed_example/
OUT=$(http_proxy=$SECONDARY_HOSTNAME
    $WGET_DUMP -O /dev/null -S $URL 2>&1)
```